### PR TITLE
[CPU] Add graph register for fused_sigmoid_mul_cpu, fused_qk_gemma_rmsnorm

### PR DESCRIPTION
**Pull request by yanbing-j**
## Motivation

This PR is to change the CPU fused_sigmoid_mul path to be an in-place, no-return custom op and added the corresponding CPU graph/fake-op support of fused_sigmoid_mul and fused_qk_gemma_rmsnorm.
## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review and Merge Process

1. Ping Merge Oncalls to start the process. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - Common commands include `/tag-and-rerun-ci`, `/tag-run-ci-label`, `/rerun-failed-ci`
4. After green CI and required approvals, ask Merge Oncalls or people with Write permission to merge the PR.

---
### CI States

Latest PR Test (Base): :x: [Run #32456106115](https://github.com/sgl-project/sglang/actions/runs/32456106115)
Latest PR Test (Extra): :x: [Run #32456105784](https://github.com/sgl-project/sglang/actions/runs/32456105784)
Latest PR Test (AMD ROCm 7.2): :x: [Run #32456106062](https://github.com/sgl-project/sglang/actions/runs/32456106062)

### CODE REVIEW

File: python/sglang/srt/models/qwen3_5.py [hunk 1]
```diff
@@ -190,7 +190,12 @@ def _maybe_enable_silu_fp4_quant_fusion(mlp: nn.Module) -> None:
     )
 
 if _is_cpu:
-    fused_sigmoid_mul = torch.ops.sgl_kernel.fused_sigmoid_mul_cpu
+    _fused_sigmoid_mul_cpu = torch.ops.sgl_kernel.fused_sigmoid_mul_cpu
+
+    def fused_sigmoid_mul(x, gate, inplace=True):
+        _fused_sigmoid_mul_cpu(x, gate)
+        return x
```
Comment (mingfeima) [inline python/sglang/srt/models/qwen3_5.py:199]: the CPU wrapper always mutates x and ignores inplace=False, unlike the Triton fused_sigmoid_mul API. Production always passes inplace=True, but please either clone when inplace=False or assert inplace so the contract is explicit.

**Review by mingfeima — CHANGES_REQUESTED** [review]

File: python/sglang/srt/models/qwen3_5.py (hunk 1 above)
Comment (yanbing-j) [reply python/sglang/srt/models/qwen3_5.py:199]: Updated.

**Review by mingfeima — APPROVED** [review]
